.circleci/config.yml

```diff
@@ -206,11 +222,17 @@ workflows:
         filters:
           branches:
             ignore: master
-    - test:
+    - e2e-preview:
+        requires:
+          - preview
         filters:
-          branches:
-            ignore: master
+            branches:
+              ignore: master
+    - test:
+       filters:
+         branches:
+           ignore: master
```

Comment:
I would recommend against this ignore master on -test because currently the master and the PR might not using same code.

Reply:
we could consider a pull merge before running the tests (or is that already happening?) 
to ensure the PR test is equivalent to master test.
and are we sure that the tests are rerun on every merge on master to retest taht

I think we probably don't want that because then PR would run very often?
In other words, I think we are better of when running tests also on master. There is not much additional cost involved to do so. (I think formally we should block the release on it then, what do you think?)

Reply:
i did not make any changes here , see master